src/bindings/python/src/pyopenvino/graph/types/element_type.cpp

```diff
@@ -31,7 +31,7 @@ void regclass_graph_Type(py::module m) {
             :rtype: ov.Type
         )");
 
-    type.attr("undefined") = ov::element::undefined;
+    type.attr("undefined") = ov::element::dynamic;
```

Comment:
Did you check that deprecation warning is also thrown for Python API?

Reply:
Looks like `ov::element::undefined` is not exposed in js api yet